### DISCUSSION TITLE
Change default browser

### DISCUSSION THREAD

**Discussion by klammbueddel** [ankitpokhrel/jira-cli#248] [Q&A] [answered]
Great tool, just just stumbled across it and fell in love. 

I am using it in WSL. I tried a lot to change the default browser (update-alternatives --config x-www-browser, setting BROWSER variable), but still Google Chrome within WSL is started. I'd like to open issues in Firefox of my host which works fine with wslview. 

How can I tell jira-cli to use wslview as default browser?

Thank you for this awesome tool!

**Comment by ankitpokhrel** [comment, marked as answer]
@klammbueddel the pkg I am using [pkg/browser](https://github.com/pkg/browser) doesn't seem to respect `BROWSER` env. I will see if I can do a quick fix around this. Meanwhile I am converting this discussion to a bug.
